src/tests/ftest/util/test_utils_container.py

```diff
@@ -920,6 +907,27 @@ def punch_records(self, indices, punch_dkey=True):
         # Return the number of punched objects
         return count
 
+    @fail_on(CommandFailure)
+    def check(self, *args, **kwargs):
+        """Check object consistency by calling daos container check.
+
+        Args:
+            args (tuple, optional): positional arguments to DaosCommand.container_check
+            kwargs (dict, optional): named arguments to DaosCommand.container_check
+
+        Returns:
+            str: JSON output of daos container check.
+
+        Raises:
+            DaosTestError: if params are invalid
+            CommandFailure: Raised from the daos command call.
+
+        """
+        if not self.daos:
+            raise DaosTestError("Undefined daos command")
```

Comment:
@phender Thoughts on removing these checks now? Nearly all tests have `container.daos` defined now that they use `self.get_container()`. The remaining manual instantiations of `TestContainer` are few and generally API tests that should eventually move to unit tests anyway